types/module/module.go

```diff
@@ -212,6 +213,11 @@ type EndBlockAppModule interface {
 	EndBlock(sdk.Context, abci.RequestEndBlock) []abci.ValidatorUpdate
 }
 
+type HasABCIEndblock interface {
```

Comment:
Do we have somewhere something that explains the difference between:

HasABCIEndblock, EndBlockAppModule and HasEndBlocker?

Those 3 extension interfaces doing seemingly the same thing (I know they don't but still) could get confusing.

Reply:
the one above gets removed in another pr. Left a todo for when we do docs on core api